applications/tari_validator_node/tests/utils/wallet_daemon_cli.rs

```diff
@@ -0,0 +1,111 @@
+//   disclaimer.
+//
+//   2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the
+//   following disclaimer in the documentation and/or other materials provided with the distribution.
+//
+//   3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote
+//   products derived from this software without specific prior written permission.
+//
+//   THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+//   INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+//   DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+//   SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+//   SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+//   WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+//   USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+use base64;
+use serde::Serialize;
+use tari_crypto::{
+    ristretto::{RistrettoPublicKey, RistrettoSecretKey},
+    signatures::CommitmentSignature,
+    tari_utilities::ByteArray,
+};
+use tari_wallet_daemon_client::{
+    types::{AccountsCreateRequest, ClaimBurnRequest, ClaimBurnResponse},
+    WalletDaemonClient,
+};
+
+use super::{validator_node_cli::get_key_manager, wallet_daemon::get_walletd_client};
+use crate::TariWorld;
+
+pub async fn claim_burn(
+    world: &mut TariWorld,
+    account_name: String,
+    commitment: Vec<u8>,
+    range_proof: Vec<u8>,
+    ownership_proof: CommitmentSignature<RistrettoPublicKey, RistrettoSecretKey>,
+    reciprocal_claim_public_key: RistrettoPublicKey,
+    wallet_daemon_name: String,
+) -> ClaimBurnResponse {
+    #[derive(Serialize)]
+    struct OwnershipProof {
+        public_nonce: String,
+        u: String,
+        v: String,
+    }
+
+    #[derive(Serialize)]
+    struct ClaimValue {
+        commitment: String,
+        ownership_proof: OwnershipProof,
+        reciprocal_claim_public_key: String,
+        range_proof: String,
+    }
+
+    let ownership_proof = OwnershipProof {
+        public_nonce: base64::encode(ownership_proof.public_nonce().as_bytes()),
+        u: base64::encode(ownership_proof.u().as_bytes()),
+        v: base64::encode(ownership_proof.v().as_bytes()),
+    };
+
+    let value = ClaimValue {
+        commitment: base64::encode(commitment.as_bytes()),
+        ownership_proof,
+        reciprocal_claim_public_key: base64::encode(reciprocal_claim_public_key.as_bytes()),
+        range_proof: base64::encode(range_proof.as_bytes()),
+    };
+
+    let claim_proof = serde_json::to_value(value).unwrap();
```

Comment:
nit: feel free to ignore, can also do this to reduce lines of code, though the way you did it is also :100: 
```suggestion
  let claim_burn_request = ClaimBurnRequest {
        account: account_address,
        claim_proof: json!({
            "commitment": base64::encode(commitment.as_bytes()),
            "ownership_proof":{
                "public_nonce": base64::encode(ownership_proof.public_nonce().as_bytes()),
                "u": base64::encode(ownership_proof.u().as_bytes()),
                "v": base64::encode(ownership_proof.v().as_bytes()),
            },
            "reciprocal_claim_public_key": base64::encode(reciprocal_claim_public_key.as_bytes()),
            "range_proof": base64::encode(range_proof.as_bytes()),
        }),
        fee: 1,
    };

```

Reply:
Awesome, this is simpler ! Refactored :)